src/main/java/mops/termine2/controller/formular/ErgebnisForm.java

```diff
@@ -0,0 +1,56 @@
+package mops.termine2.controller.formular;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import mops.termine2.enums.Antwort;
+import mops.termine2.models.Terminfindung;
+import mops.termine2.models.TerminfindungAntwort;
+import mops.termine2.util.LocalDateTimeManager;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class ErgebnisForm {
+	
+	List<LocalDateTime> termine = new ArrayList<>();
+	
+	List<Integer> anzahlStimmenJa = new ArrayList<>();
+	
+	List<Integer> anzahlStimmenVielleicht = new ArrayList<>();
+	
+	List<Integer> anzahlStimmenNein = new ArrayList<>();
+	
+	public ErgebnisForm(List<TerminfindungAntwort> antworten, Terminfindung terminfindung) {
+		termine = terminfindung.getVorschlaege();
+		LocalDateTimeManager.sortTermine(termine);
+		for (int i = 0; i < termine.size(); i++) {
```

Comment:
Hier wäre eine for each Schleife schöner.

Reply:
Geändert